src/Admin/Content/AdminPayload.php

```diff
@@ -75,6 +75,9 @@ public function __invoke(Document $document, Request $request)
         $document->payload['extensions'] = $this->extensions->getExtensions()->toArray();
 
         $document->payload['displayNameDrivers'] = array_keys($this->container->make('flarum.user.display_name.supported_drivers'));
+        $document->payload['slugDrivers'] = array_map(function ($resourceDrivers) {
```

Comment:
Similarly to the internal discussion about moving backend pre-processing of extensions to a method in `AdminPayload`, this chunk of code would also be a good candidate for a `serializeSlugDrivers` method.